### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by JesseVanEvert**
Tanzeel, Brandon and Mahedi review this README, and let me know if you want anything to be changed.
Some of the markup did not work as I thought, so that will be changed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## contributions
+
+- Brandon: 
+  - Frontend development with vue.js and bootstrap;
+  - Development student and class service;
+  - Attempt deployment openshift and minikube + kubernetes.
+  
+- Tanzeel:
+  - Development student and class service;
+  - Attempt deployment openshift and minikube + kubernetes.
```
Comment (htanzeel01) [inline README.md]: - Tanzeel:
  - Development student and class service;
  - Attempt deployment openshift and minikube + kubernetes.
  - Frontend development with vue.js and bootstrap;
  - implemented rabbitmq for classroom service
  - used swagger for api documentation 
  - did frontend testing/ allocated proper ports to bootstrap
  - helped with configuring sendgrid
 
- Mahedi:
  - Development message service;
  - Attempt deployment openshift and minikube + kubernetes.
  - Frontend development with vue.js and bootstrap;
  - implemented rabbitmq for service for consuming messages
  - used sendgrid implementation to send emails to students
  - tested with postman
  - tested frontend demo/ helped prepping demo 
  - helped with docker implementation on python

**Review by htanzeel01 — CHANGES_REQUESTED** [review]
Tanzeel:

Development student and class service;
Attempt deployment openshift and minikube + kubernetes.
Frontend development with vue.js and bootstrap;
implemented rabbitmq for classroom service
used swagger for api documentation
did frontend testing/ allocated proper ports to bootstrap
helped with configuring sendgrid
Mahedi:

Development message service;
Attempt deployment openshift and minikube + kubernetes.
Frontend development with vue.js and bootstrap;
implemented rabbitmq for service for consuming messages
used sendgrid implementation to send emails to students
tested with postman
tested frontend demo/ helped prepping demo
helped with docker implementation on python

File: README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+ 
+- Mahedi:
+  - Development message service;
+  - Attempt deployment openshift and minikube + kubernetes.
```
Comment (mridul-max) [inline README.md]: - Development message service;
- Attempt deployment openshift and minikube + kubernetes.
- Frontend development with vue.js and bootstrap;
- implemented rabbitmq for service for consuming messages
- used sendgrid implementation to send emails to students
- tested with postman
- tested frontend demo/ helped prepping demo
- helped with docker implementation on python

File: README.md (hunk 1 above)
Comment (mridul-max) [reply README.md:23]: I agreed

File: README.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+The architecture design can be found in the wiki, or in the master branch -> documents
+*
+
+## contributions
+
+- Brandon: 
+  - Frontend development with vue.js and bootstrap;
+  - Development student and class service;
+  - Attempt deployment openshift and minikube + kubernetes.
+
```
Comment (Taybl98) [inline README.md:24]: Aided in Frontend Containerization

Assisted in Python RabbitMQ implementation

Comment (Taybl98) [reply README.md:24]: Frontend Preperation for testing

**Review by Taybl98 — DISMISSED** [review]

**Review by mridul-max — DISMISSED** [review]

**Comment by JesseVanEvert** [comment]
I have several remarks and questions on/about the comments you guys provided.
To remind you guys you'll have to be very specific about what you've done, because it will be hard/impossible to deduce from the commits.

Tanzeel:
- You did not implement rabbitmq in the classroom service, but in the student service
- Can you be more specific what you did for the development of the frontend
- Helped with implementing sendgrind? What does this mean? Should I also list everyting I helped out with?
- I like the swagger addition, did you implement them for both the classroom and student API?

Mahedi:
- What do you mean by: "helped with docker implementation on python"
- What did you test with Postman? Message service I assume?
- Other additions seem fine

Brandon:
- What do you mean by: "Aided in Frontend Containerization", "Assisted in Python RabbitMQ implementation" and "
Frontend Preperation for testing" elaborate and be specific, also I would prefer to read about the actual code you've written.

**Comment by htanzeel01** [comment]
I have several remarks and questions on/about the comments you guys provided.
To remind you guys you'll have to be very specific about what you've done, because it will be hard/impossible to deduce from the commits.

Tanzeel:

You did not implement rabbitmq in the classroom service, but in the student service (I did implement it for student service which used class service regardless. You changed it class service which was a choice) 
Can you be more specific what you did for the development of the frontend (i made my own frontend and even did the testing for it all brandon did was put my frontend code together he did not create my frontend also I was the one who tested his part as well)
Helped with implementing sendgrind? What does this mean? Should I also list everyting I helped out with? (Dude you can list whatever you want and I helped out in implementing using the message to send it to the correct email. Its called collaboration) 
I like the swagger addition, did you implement them for both the classroom and student API?(I was the 1 who implemented everything in python. I helped brandon out with everything he did)

**Pull request merged**